styles.css

```
@@ -0,0 +1,313 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:wght@300&family=Roboto:wght@100&display=swap');
+
+/*CSS reset*/
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+:root {
+    --purple: #5722CD;
+    --blue: #4845E4;
+    --pink: #ED4385;
+    --black: #0C0C0C;
+    --purplelight: #F2F0FF;
+    --greylight: #F3F6F9;
+    --grey: #7A7A7A;
+    --white: #ffffff;
+}
+
+
+header {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    /*background-color: var(--blue);*/
+    /*background: rgb(72,69,228);*/
+    background: linear-gradient(90deg, rgba(72,69,228,1) 44%, rgba(49,42,157,1) 100%, rgba(0,212,255,1) 100%);
+    color: var(--white);
+    padding: 10px 10px 0 10px;
+}
+
+.navigation {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    color: var(--white);
+}
+
+.nlink {
+    font-size: 12px;
+    color: var(--white);
+}
+
+.navigation ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    flex-basis: 300px;
+    flex-grow: 0;
+    flex-shrink: 1;
+    list-style: none;
+    color: var(--white);
+}
+
+.center-title {
+    text-align: center;
+    margin: 50px 350px 100px 350px;
+}
+
+.second-title {
+    padding: 20px;
+}
+
+.button {
+    position: center;
+    text-transform: uppercase;
+    background-color: var(--pink);
+    color: var(--white);
+    border: none;
+    border-radius: 50px;
+    font-size: 6px;
+    padding: 15px 32px;
+    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2), 0 6px 20px 0 rgba(0,0,0,0.19);
+}
+
+#intro-sec {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    background-color: var(--white);
+    padding: 10px;
+}
+
+.introduction {
+    display: flex;
+    flex-basis: 1400px;
+    flex-direction: column;
+    justify-content: center;
+    text-align: center;
+    color: var(--purple);
+    font-weight: bold;
+    margin: 50px 350px 50px 350px;
```

Hier geldt eigenlijk hetzelfde
```suggestion
    margin: 50px;
```